cartoframes/data/observatory/enrichment/enrichment.py

```diff
@@ -318,6 +321,34 @@ def enrich_polygons(self, dataframe, variables, geom_col=None, filters={}, aggre
                 enrichment = Enrichment()
                 cdf_enrich = enrichment.enrich_polygons(df, variables, aggregation=aggregation)
 
+            Enrich a polygons dataframe using several aggregation methods for a variable:
+
+            .. code::
```

Comment:
Will https://github.com/CartoDB/cartoframes/pull/1409 be merged into this PR first? There are some larger docs changes in that PR that will change how examples are displayed.

Reply:
We will have a conflict to solve in the merge in any case